### DISCUSSION TITLE
Anyone had luck with a7m3 (a7iii)

### DISCUSSION THREAD

**Discussion by ccc9090** [frank26080115/alpha-fairy#26] [Q&A]
I am able to connect and do a few things but the focus control doesn't work at all.

Anyone had any luck with this?

**Comment by frank26080115** [comment]
Can you confirm that the camera is in MF mode, and the connection is done through the newer PTP protocol? (see instructions, where it says what connection method uses PTP protocol)

Unfortunately, I don't see the A7M3 listed as being supported by the PTP protocol, https://support.d-imaging.sony.co.jp/app/sdk/en/index.html

The older JSON protocol doesn't support manual focus adjustments.

The alpha-fairy menu should have given you an error message saying "Unsupported Camera" if you tried to do a focus pull

**Comment by tao-j** [comment]
@ccc9090  A7M3 does not work with PTP so no manual control. But if we figure out how to make BT work #31 , then you can do manual focus.
